pkg/manager/member/tidb_member_manager.go

```diff
@@ -622,6 +622,7 @@ func getNewTiDBSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 					SecurityContext: &corev1.SecurityContext{
 						Privileged: &privileged,
 					},
+					Resources: tc.GetHelperResourceRequirements(),
```

Comment:
Prefer to inherit TiDBSpec's resource instead of composing `ResourceRequrements` into `HelperSpec`. We inherit the resource requirements from `TiDBSpec` for initializing containers also ensure the Pod won't exceed the resource limit.

Reply:
I guess `HelperSpec` is a common setting for tidb/tikv initContainer, so I think `ResourceRequrements` should be setting in `HelpSpec`. Am I right ?
https://github.com/shonge/tidb-operator/blob/027c663eba23e8eff1a368de092aeea863670edc/pkg/manager/member/tidb_member_manager.go#L616

Reply:
Found this document:
https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#resources
I will update like this:
tidb inticontainer resourceRequrement = tidb container resourceRequrement 
tikv inticontainer resourceRequrement = tikv container resourceRequrement 
Is that OK?

Reply:
I think it's OK, the init container is used to do some simple tasks.
How do you think? @DanielZhangQD

Reply:
Update